src/sql/workbench/contrib/executionPlan/browser/executionPlan.ts

```diff
@@ -59,6 +59,12 @@ export interface InternalExecutionPlanEdge extends azdata.executionPlan.Executio
 	id?: string;
 }
 
+export enum BadgeType {
+	WARNING = 0,
+	CRITICALWARNING = 1,
```

Comment:
I would say it would be okay to use underscores to make `CRITICAL_WARNING` easier to read, but not a big deal.